crates/driver/src/domain/eth/mod.rs

```diff
@@ -322,6 +322,19 @@ pub struct Tx {
     pub access_list: AccessList,
 }
 
+impl std::fmt::Debug for Tx {
```

Comment:
Is this what we intend to send to solvers? I would honestly just sent them a json with very explicit fields (from, to, value, data, access_lists)

Reply:
No, this is just added because we enriched the errors with this struct to be printed to logs and I wanted the `input` file to be logged as hex.
What we send to solvers will be defined by `dto` object that will contain raw `Tx` struct available to take whatever it wants.

Reply:
Even for developers reading this log, I think it'd be simpler to just have {from: ..., to: ..., value: ... calldata: ...} instead of defining our own transaction notation.

Reply:
Agreed. Fixed.